README.md

```diff
@@ -1,8 +1,14 @@
 # FastStats
 
-Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/fast_stats`. To experiment with that code, run `bin/console` for an interactive prompt.
+| Travis CI|
+|----------|
+| [![Build Status](https://travis-ci.org/kyle-rader/fast_stats.svg?branch=master)](https://travis-ci.org/kyle-rader/fast_stats) |
+
+Welcome to FastStats!
+This gem is for computing stats in an efficient manner. Right now it just manages a collection of arithmetic and geometric means to help reduce the [data clump code smell](https://sourcemaking.com/refactoring/smells/data-clumps).
```

Comment:
This looks fine when the Markdown is rendered, but it's difficult to read in an editor (especially on smaller screens) as it extends out 229 columns. Limiting the column width of your `README.md` to something like 80 or even 120 columns could help with this.

![long-lines-are-loooong](https://user-images.githubusercontent.com/3084059/36636879-0339c4c2-1984-11e8-8243-cae841b3d0ba.png)